operator/controllers/csiscaleoperator_controller.go

```diff
@@ -2397,11 +2397,11 @@ func validateVolNamePrefix(key string, value string, data map[string]string, inv
 	logger := csiLog.WithName("validateVolNamePrefix")
 	logger.Info("Validating volume name prefix input ", "volNamePrefix", value)
 
-	if len(value) > 2 && len(value) < 6 {
+	if len(value) > 2 && len(value) < 6 && !strings.ContainsAny(value, " ") {
```

Comment:
@deeghuge @Jainbrt Here we thinking to add check for special characters as fileset creation fails , please let us know your opinion on same 
`Internal desc = unable to create fileset [$$-2c503dae-585d-4ac9-9c9e-dc04ba92d4cd] in filesystem [fs1]. Error: rpc error: code = Internal desc = remote call failed with response &{{409 Input validation failed: EFSSP1101C CLI: The value "$$-2c503dae-585d-4ac9-9c9e-dc04ba92d4cd" specified for "filesetName" is invalid. The name shall not contain whitespace or any of '#,;:/?&*"()[]'.} []}: POST request`

Reply:
addressed the comment